tests/app/submitter/test_submitter.py

```diff
@@ -181,15 +181,15 @@ def test_send_message():
             assert published is True
 
     @staticmethod
-    def test_send_message_adds_tx_id_metadata():
+    def test_send_message_adds_metadata():
         # Given
         with patch('app.submitter.submitter.storage.Client') as client:
             # When
             submitter = GCSSubmitter(project_id='test_project', bucket_name='test_bucket')
-            submitter.send_message(message={'test_data'}, case_id='123', tx_id='456')
+            submitter.send_message(message={'test_data'}, tx_id='123', case_id='456')
```

Comment:
You've swapped the values here, but not in the assert.

Reply:
Sorted